### PR TITLE
story/EUI-5345 Cancelled hearing

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5345

### Change description ###
View details of Hearings from Hearings tab- Cancelled hearing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/cases/components/case-hearings-list/case-hearings-list.component.ts [hunk 1]
```diff
@@ -77,6 +77,9 @@ export class CaseHearingsListComponent implements OnInit {
     if (hearing.exuiDisplayStatus === EXUIDisplayStatusEnum.CANCELLATION_REQUESTED) {
       this.hearingStore.dispatch(new fromHearingStore.LoadHearingRequest(hearing.hearingID));
       this.router.navigate(['/', 'hearings', 'request', 'hearing-cancellation-summary']);
+    } else if (hearing.exuiDisplayStatus === EXUIDisplayStatusEnum.CANCELLED) {
+      this.hearingStore.dispatch(new fromHearingStore.LoadHearingRequest(hearing.hearingID));
+      this.router.navigate(['/', 'hearings', 'request', 'hearing-cancelled-summary']);
```
Comment (rayliangatos) [inline src/cases/components/case-hearings-list/case-hearings-list.component.ts:82]: If this for view details, could we move this to hearings/view/hearing-cancelled-summary, as it doesn't request page flow and different journey, same as the above one hearings/view/hearing-cancellation-summary

File: src/hearings/containers/request-hearing/hearing-cancelled-summary/hearing-cancelled-summary.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { HEARING_CANCELLED_SUMMARY_TEMPLATE } from 'src/hearings/templates/hearing-cancelled-summary.template';
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/hearing-cancelled-summary/hearing-cancelled-summary.component.ts:3]: src

File: src/hearings/containers/request-hearing/hearing-cancelled-summary/hearing-cancelled-summary.component.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import { ACTION, Mode } from '../../../models/hearings.enum';
+import { HearingsService } from '../../../services/hearings.service';
+import * as fromHearingStore from '../../../store';
+import { RequestHearingPageFlow } from '../request-hearing.page.flow';
+
+@Component({
+  selector: 'exui-hearing-cancelled-summary',
+  templateUrl: './hearing-cancelled-summary.component.html',
+})
+export class HearingCancelledSummaryComponent extends RequestHearingPageFlow implements OnDestroy {
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/hearing-cancelled-summary/hearing-cancelled-summary.component.ts:13]: Is that better if we don't use this RequestHearingPageFlow? Use the back route directly on the parent summary page. Same as another component HearingCancellationSummaryComponent.

File: api/prd/caseFlag/caseFlag.mock.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ import {CASE_FLAG_REFERENCE_VALUES} from './data/caseFlagReference.mock.data';
 export const init = () => {
   const mock: MockAdapter = HttpMockAdapter.getInstance();
 
-  const getCaseFlagRefDataUrl = /http:\/\/rd-professional-api-aat.service.core-compute-aat.internal\/caseflagrefdata/;
+  const getCaseFlagRefDataUrl = /caseflagrefdata/;
```
Comment (kuda1992) [inline api/prd/caseFlag/caseFlag.mock.ts:9]: Why did we remove the origin ulr

**Review by kuda1992 — APPROVED** [review]
LGTM

**Pull request merged**